app/src/main/java/com/lagradost/cloudstream3/ui/result/SelectAdaptor.kt

```diff
@@ -45,19 +45,9 @@ class SelectAdaptor(val callback: (Any) -> Unit) : RecyclerView.Adapter<Recycler
         if(newIndex == selectedIndex) return
         val oldIndex = selectedIndex
         selectedIndex = newIndex
-        recyclerView.apply {
```

Comment:
This failed on edgecases in the recyclerview.
Although not as efficient it is easier to only update the items for guaranteed results.